src/components/UI/index.ts

```diff
@@ -0,0 +1,6 @@
+import AlertBanner from "./AlertBanner"
+import Button from "./Button"
+import Card from "./Card"
+import Loading from "./Loading"
+
+export { AlertBanner, Button, Card, Loading }
```

Comment:
```suggestion
export { default as AlertBanner } from "./AlertBanner"
export { default as Button } from "./Button"
export { default as Card } from "./Card"
export { default as Loading } from "./Loading"
```

이 부분은 다시내보내기에 이런 방법도 있다는 것을 공유드리기 위해 코멘트 남깁니다.(권장 아님)

Reply:
이런 방법도 있었네요! 참고하겠습니다